#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_warnings.dpatch by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
##
## DP: Enable more compile warnings

@DPATCH@
Index: roadmap/src/Makefile
===================================================================
RCS file: /cvsroot/roadmap/roadmap/src/Makefile,v
retrieving revision 1.47
diff -u -3 -p -u -r1.47 Makefile
--- roadmap/src/Makefile	6 Dec 2004 23:15:38 -0000	1.47
+++ roadmap/src/Makefile	31 Jan 2005 21:50:21 -0000
@@ -1,5 +1,5 @@
 
-WARNFLAGS = -W -Wall -Wno-unused-parameter -Wcast-align -Wpointer-arith -Wreturn-type -Wsign-compare
+WARNFLAGS = -W -Wall -Wcast-align -Wmissing-prototypes -Wstrict-prototypes -Wpointer-arith -Wreturn-type -Wconversion -Wsign-compare -Wcast-qual -Wmissing-declarations
 
 ifeq ($(MODE),DEBUG)
 # Memory leak detection using mtrace:
